File: frameworks/Java/fair/java-kora1-jdbc-repository/src/main/java/io/techempower/benchmark/kora/model/Fortune.java
package io.techempower.benchmark.kora.model;

import ru.tinkoff.kora.database.common.annotation.Column;
import ru.tinkoff.kora.database.common.annotation.Id;
import ru.tinkoff.kora.database.jdbc.EntityJdbc;

@EntityJdbc
public record Fortune(@Id @Column("id") int id,
                      @Column("message") String message) {
}

File: frameworks/Java/fair/kotlin-kora1-vertx-repository-suspend/src/main/kotlin/io/techempower/benchmark/kora/model/Fortune.kt
package io.techempower.benchmark.kora.model

import ru.tinkoff.kora.database.common.annotation.Column
import ru.tinkoff.kora.database.common.annotation.Id

data class Fortune(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("message") val message: String,
)
